a_perfect_dish_to_share.py

```diff
@@ -0,0 +1,16 @@
+def get_perfect_divided(upper_bound=10000):
+    """
+    The function checks each number from 1 to the upper_bound if it is a perfect number to divide. With the help of
+    a generator and a sum function.
+    :param upper_bound: A number that defines the upper bound of the numbers that returns from the generator,
```

Comment:
Try to use generators so the function will be able to run infinitely but without raising time error.